edalize/libero.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import os.path
+import os
+import platform
+import subprocess
+import re
+import xml.etree.ElementTree as ET
+from functools import partial
+from edalize.edatool import Edatool
```

Comment:
logging and Edatool are the only imports that are used in the code. Remove the rest

Reply:
Great, I'm too used to Go where imports not used generate compile errors or VSCode plugin manages import automatically. :)
Cleaned.